parts/windows/csecmd.ps1

```diff
@@ -1,21 +1,14 @@
 powershell.exe -ExecutionPolicy Unrestricted -command \"
 $inputFile = '%SYSTEMDRIVE%\AzureData\CustomData.bin';
 $outputFile = '%SYSTEMDRIVE%\AzureData\CustomDataSetupScript.ps1';
-if (!(Test-Path $inputFile)) { throw 'ExitCode: |49|, Output: |WINDOWS_CSE_ERROR_NO_CUSTOM_DATA_BIN|, Error: |C:\AzureData\CustomData.bin does not exist.|' };
+if (!(Test-Path $inputFile)) { throw 'ExitCode: |49|, Output: |WINDOWS_CSE_ERROR_NO_CUSTOM_DATA_BIN|, Error: |C:\AzureData\CustomData.bin  does not exist.|' };
```

Comment:
nit: looks like an extra space was added between `C:\AzureData\CustomData.bin` and  `does not exist.`, since we're trying to minimize size might be worth cleaning that up

Reply:
Even better, I can replace it with "$inputFile" :)